tools/hooks/scary_changes.rb

```diff
@@ -66,12 +66,32 @@ def detect_special_files
     end
   end
 
+  def detect_dropbox_conflicts
+    changes = @all.grep(/'s conflicted copy/)
```

Comment:
Should this be `@all` or `@added`? For example, I think this would block deletion of existing conflicted copies, something we probably want to allow.